### ISSUE TITLE
strange progress bar output

### ISSUE THREAD

**Issue by swails** [Amber-MD/cpptraj#154]
When I ran cpptraj with a bunch of trajectories recently, this is the output I got:

```
ACTION SETUP FOR PARM '2igd_924wat.pdb' (2 actions):
  0: [autoimage]
	Anchor molecule is 1
	The following molecules are fixed to anchor: 2 3 4
	932 molecules are mobile.
  1: [grid out restrained_density.dx 40 40 40 1 1 1 :WAT@O normdensity]
	Mask [:WAT@O] corresponds to 924 atoms.
----- 2igd_924wat.md1.nc (1-5000, 1) -----
 0% 10% 20% 
----- 2igd_924wat.md2.nc (1-5000, 1) -----
30% 40% 
----- 2igd_924wat.md3.nc (1-1000, 1) -----
50% 
----- 2igd_924wat.md4.nc (1-1000, 1) -----

----- 2igd_924wat.md5.nc (1-1000, 1) -----
60% 
----- 2igd_924wat.md6.nc (1-1000, 1) -----

----- 2igd_924wat.md7.nc (1-1000, 1) -----
70% 
----- 2igd_924wat.md8.nc (1-1000, 1) -----

----- 2igd_924wat.md9.nc (1-1000, 1) -----
80% 
----- 2igd_924wat.md10.nc (1-1000, 1) -----

----- 2igd_924wat.md11.nc (1-1000, 1) -----
90% 
----- 2igd_924wat.md12.nc (1-1000, 1) -----

----- 2igd_924wat.md13.nc (1-1000, 1) -----
100% Complete.

Read 21000 frames and processed 21000 frames.
TIME: Trajectory processing: 7.7336 s
TIME: Avg. throughput= 2715.4071 frames / second.
```

In the past, each trajectory would get its own progress bar.  Is this change intended?

**Comment by drroe** [comment]
@swails Yeah, this was implemented during #57 - I moved the progress bar out of the Trajectory classes and was thinking "well, instead of having the progress bar be on a trajectory basis what if it reflected how much of the total run was complete?". Both can be interesting. One way that I was thinking could be the best of both worlds would be to adjust the frequency of progress bar writes based on the current number of frames (so each traj bar would still have 10 numbers written) but it hasn't been a huge priority. Any thoughts?

**Comment by swails** [comment]
It looks a little weird as-is.  I might have a loose preference for printing all of the trajectories that are going to be processed first, then having a continuous progress bar at the end.  So something that looks like

```
----- 2igd_924wat.md1.nc (1-5000, 1) -----
----- 2igd_924wat.md2.nc (1-5000, 1) -----
----- 2igd_924wat.md3.nc (1-1000, 1) -----
----- 2igd_924wat.md4.nc (1-1000, 1) -----
----- 2igd_924wat.md5.nc (1-1000, 1) -----
----- 2igd_924wat.md6.nc (1-1000, 1) -----
----- 2igd_924wat.md7.nc (1-1000, 1) -----
----- 2igd_924wat.md8.nc (1-1000, 1) -----
----- 2igd_924wat.md9.nc (1-1000, 1) -----
----- 2igd_924wat.md10.nc (1-1000, 1) -----
----- 2igd_924wat.md11.nc (1-1000, 1) -----
----- 2igd_924wat.md12.nc (1-1000, 1) -----
----- 2igd_924wat.md13.nc (1-1000, 1) -----
 0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100%
```

But then you don't know exactly which trajectory is being read at the time.

In the grand scheme of things, this is a tiny nit.  I rather preferred the old look, myself, but this approach has its advantages as well.

I'll go ahead and close this since it's not worth putting much time into IMO.
